### ISSUE TITLE
show link shortcuts on page load

### ISSUE THREAD

**Issue by whIzz2000** [philc/vimium#1931]
As a user of Vimium with voice recognition I rely on the use of link shortcuts to do the majority of page navigation. I would benefit greatly from being able to have these link shortcuts available automatically on page load rather than having to dictate a macro to issue keypress "f" each time a page loads.

I have done a quick search on Google and github to see if this issue has been referenced, apologies if I missed a relevant issue.

In summary it would be great to have the ability to automatically show link shortcuts on page load.

Thanks so much for the great work

**Comment by whIzz2000** [comment]
found a rudimentary solution using cVim with a configuration option: https://gist.github.com/whIzz2000/8176342e2a8bf0c61629

**Comment by whIzz2000** [comment]
would be interested in Vimium solutions if any are available
